From cf162aaae84935bd0a227690bc9dbed1c228f044 Mon Sep 17 00:00:00 2001
From: Min RK <benjaminrk@gmail.com>
Date: Mon, 29 Oct 2018 10:57:15 +0100
Subject: [PATCH] include stdio.h in hocdec.h

compilation with gcc >= 7 can fail with

    oc_ansi.h:109:24: error: unknown type name 'FILE'

due to stdio.h not being included.
There is a conditional include of stdio.h in hocassrt.h that may be related.
---
 src/oc/hocdec.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/oc/hocdec.h b/src/oc/hocdec.h
index fbddb3b0..b5b03e53 100644
--- a/src/oc/hocdec.h
+++ b/src/oc/hocdec.h
@@ -8,6 +8,7 @@
 extern "C" {
 #endif
 
+#include    <stdio.h>
 #include	"nrnapi.h"
 #include	"hocassrt.h"
 #include	<string.h>
-- 
2.19.1

